api/src/client/dep/consolidatedEntries.test.ts

```diff
@@ -0,0 +1,29 @@
+import { consolidatedEntries } from "@client/dep/consolidatedEntries";
+import { generateXrayRegistrationEntry } from "@test/factories";
+
+describe("consolidatedEntries", () => {
+  it("returns entries that are present in both arrays", () => {
+    const businessNameResults = [
+      generateXrayRegistrationEntry({
+        registrationNumber: "12345",
+      }),
+      generateXrayRegistrationEntry({
+        registrationNumber: "67890",
+      }),
+    ];
+
+    const addressResults = [
+      generateXrayRegistrationEntry({
+        registrationNumber: "12345",
+      }),
+      generateXrayRegistrationEntry({
+        registrationNumber: "54321",
+      }),
+    ];
+
+    const result = consolidatedEntries(addressResults, businessNameResults);
```

Comment:
[dust] Should we destructure this array and switch to testing if result is undefined? It might make TypeScript a little happier.